node/parallel/src/command.rs

```diff
@@ -36,18 +36,35 @@ use sp_runtime::traits::Block as BlockT;
 use std::{io::Write, net::SocketAddr};
 
 const CHAIN_NAME: &str = "Parallel";
+const PARA_ID: u32 = 2085;
+
+fn load_spec(id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, String> {
+    let (norm_id, para_id) = extract_parachain_id(id);
```

Comment:
the only paraid should be `ParachainInfo::paraid` so we should have put para id inside chainspec

Reply:
`ParachainInfo::paraid` is not available during the build-spec stage, now use the constant to specify para_id=2085 for $name in all `build-spec --chain=$name`